samples: applications: Use ipc_radio as default radio firmware. (#15435)
nrfconnect/sdk-nrf: samples/openthread/coap_client/README.rst

* ``multiprotocol_ble`` - Enables the Multiprotocol Bluetooth LE extension.

  .. note::

>     When building with ``multiprotocol_ble`` snippet, for the ``nrf5340dk/nrf5340/cpuapp`` build target, additional :makevar:`FILE_SUFFIX=ble` option has to be set.

Reviewer: ```suggestion
    When building with ``multiprotocol_ble`` snippet, for the ``nrf5340dk/nrf5340/cpuapp`` board target, additional :makevar:`FILE_SUFFIX=ble` option has to be set.
```
Same comment as the previous.

>     When building with the ``multiprotocol_ble`` snippet for the ``nrf5340dk/nrf5340/cpuapp`` board target, set the additional :makevar:`FILE_SUFFIX` CMake variable to ``ble``.

Reviewer 2: ```suggestion
    When building with the ``multiprotocol_ble`` snippet for the ``nrf5340dk/nrf5340/cpuapp`` board target, set the additional :makevar:`FILE_SUFFIX` CMake option to ``ble``.
    See :ref:`app_build_file_suffixes` and :ref:`cmake_options` for more information.
```

>     When building with ``multiprotocol_ble`` snippet, for the ``nrf5340dk/nrf5340/cpuapp`` build target, additional :makevar:`FILE_SUFFIX` CMake variable has to be set to ``ble``.

Reviewer: ```suggestion
    When building with the ``multiprotocol_ble`` snippet for the ``nrf5340dk/nrf5340/cpuapp`` board target, set the additional :makevar:`FILE_SUFFIX` CMake variable to ``ble``.
```
